inc/includes.php

```diff
@@ -175,3 +175,14 @@
 if (isset($_REQUEST["force_entity"]) && ($_SESSION["glpiactive_entity"] ?? -1) != $_REQUEST["force_entity"]) {
     Session::changeActiveEntities($_REQUEST["force_entity"], true);
 }
+
+// The user's current groups are stored in his session
+// If there was any change regarding groups membership and/or configuration, we
+// need to reset the data stored in his session
+$last_group_change = Config::getConfigurationValue('core', 'last_group_change');
+if (
+    isset($_SESSION['glpigroups'])
+    && ($_SESSION['glpigroups_cache_date'] ?? "") < $last_group_change
+) {
```

Comment:
IMHO, this entry should be stored in cache, not in DB:
```suggestion
$last_group_change = $GLPI_CACHE->get('last_group_change');
if (
    isset($_SESSION['glpigroups'])
    && ($_SESSION['glpigroups_cache_date'] ?? "") < $last_group_change
) {
```

For information, I just checked, and groups will not be reloaded if cache entry is not existing (no need to force its definition on install/update).
```
$ php -r 'var_dump("2022-10-11 23:42:23" < null);'
bool(false)
$ php -r 'var_dump("" < null);'
bool(false)
```

Reply:
Done 6f7e20cf94.